nwb_conversion_tools/nwbconverter.py

```diff
@@ -113,6 +114,7 @@ def validate_conversion_options(self, conversion_options):
     def run_conversion(
         self,
         metadata: Optional[dict] = None,
+        session_start_time: Optional[datetime] = None,
```

Comment:
why add this as an additional arg?

Reply:
Fair point, suppose it should just be an extra arg for the standalone write functions in utils